### PR TITLE
Fixing/ld related issues

### LINKED ISSUE #564: Examination and consultation form(Malaria Test)
- [x] if malaria test not conducted add a text panel which state" why was the test not conducted"

### LINKED ISSUE #565: Triage form
- [ ] if SPO2 is not filled ,the form can be saved but its displayed in yellow tick (triage is pending)

### LINKED ISSUE #567: immediately newborn care form
- [x] if the **newborn status** is still birth, also show the **sex** question

### PR DESCRIPTION
**Pull request by zk497**
Closes #564 , Closes #565 , Closes #567

### CODE REVIEW

File: opensrp-chw-hf/src/main/assets/json.form/labour_and_delivery_registration_anc_clinic_findings.json [hunk 1]
```diff
@@ -545,6 +545,24 @@
...
+          "value": "true",
+          "err": "Please answer this question"
+        },
+        "relevance": {
+          "step1:malaria": {
+            "type": "string",
+            "ex": "equalTo(., \"test_not_conducted\")"
+          }
+        }
+      },
```
Comment (cozej4) [inline opensrp-chw-hf/src/main/assets/json.form/labour_and_delivery_registration_anc_clinic_findings.json]: This is not required here

File: opensrp-chw-hf/src/main/java/org/smartregister/chw/hf/interactor/LDActiveManagementStageActivityInteractor.java [hunk 2]
```diff
@@ -197,7 +197,7 @@ public String evaluateSubTitle() {
         public BaseLDVisitAction.Status evaluateStatusOnPayload() {
             if (StringUtils.isNotBlank(placenta_and_membrane_expulsion) && StringUtils.isNotBlank(estimated_blood_loss)) {
                 return BaseLDVisitAction.Status.COMPLETED;
-            } else if (StringUtils.isBlank(estimated_blood_loss)) {
+            } else if (StringUtils.isNotBlank(placenta_and_membrane_expulsion) && StringUtils.isBlank(estimated_blood_loss)) {
```
Comment (cozej4) [inline opensrp-chw-hf/src/main/java/org/smartregister/chw/hf/interactor/LDActiveManagementStageActivityInteractor.java]: Refactor this condition to

`StringUtils.isNotBlank(placenta_and_membrane_expulsion) || StringUtils.isNotBlank(estimated_blood_loss)`

**Review by cozej4 — CHANGES_REQUESTED** [review]

File: opensrp-chw-hf/src/main/assets/json.form/labour_and_delivery_registration_anc_clinic_findings.json (hunk 1 above)
Comment (zk497) [reply opensrp-chw-hf/src/main/assets/json.form/labour_and_delivery_registration_anc_clinic_findings.json]: @cozej4 Okay, got it!

File: opensrp-chw-hf/src/main/java/org/smartregister/chw/hf/interactor/LDActiveManagementStageActivityInteractor.java (hunk 2 above)
Comment (zk497) [reply opensrp-chw-hf/src/main/java/org/smartregister/chw/hf/interactor/LDActiveManagementStageActivityInteractor.java]: @cozej4 Alright, on it!

**Review by cozej4 — APPROVED** [review]

**Pull request merged**